src/store/reducers/list.js

```diff
@@ -0,0 +1,40 @@
+import { ALL_LIST, ADD_LIST, EDIT_LIST, DELETE_LIST } from "../types";
+const initialState = {
+  lists: [],
+  todosInLists: [],
+};
+
+export default function (state = initialState, actions) {
+  switch (actions.type) {
+    case ALL_LIST:
+      const { lists } = actions.payload;
+      return {
+        ...state,
+        lists: lists,
+      };
+    case ADD_LIST:
+      const { _id, name } = actions.payload;
+      const newList = { _id: _id, name };
```

Comment:
as you can see here you've declared 3 variables here without any proper purpose
you could also do it like this
return {
        ...state,
        lists: state.lists.concat({...actions.payload}),
      };
      
      btw i think concat works on arrays!!

Reply:
yes this is working, done !